packages/react-components/lib/dispensers/dispenser-accordion.tsx

```diff
@@ -91,20 +114,21 @@ export const DispenserAccordion = React.forwardRef(
             variant: statusLabelClass ? 'normal' : 'unknown',
           }}
         />
-        {dispenserState ? (
-          <ItemAccordionDetails>
-            <DispenserInfo dispenser={dispenserState} />
-          </ItemAccordionDetails>
-        ) : (
-          <React.Fragment>
-            <Divider />
-            <Typography className={classes.typography} variant="body1">
-              The state of <b>{dispenser}</b> dispenser is unknown. Please check if{' '}
-              <b>{dispenser} </b>
-              is working properly.
-            </Typography>
-          </React.Fragment>
-        )}
+        <ItemAccordionDetails>
+          <ErrorOverlay
+            errorMsg={
+              !dispenserState
+                ? 'Dispenser is not sending states. Please check if it is working properly.'
+                : undefined
```

Comment:
A question to follow the logic. If a dispenser has a state the error message it's undefined first, and in a moment the backend stops sending the state to the UI, so now the errorMsg is 'Dispenser is not sending states. Please check if it is working properly.' If the backend starts sending the state again you will assign the errorMsg as undefined again, right?

nit: If that the case I think we should be using null. If not,  you can disregard this comment